packages/core-web/src/media/browser/controls/airplay.ts

```diff
@@ -0,0 +1,34 @@
+interface AirPlayHTMLMediaElement extends HTMLMediaElement {
```

Comment:
There's a global definition of `HTMLVideoElement` which we can move this to: https://github.com/livepeer/livepeer.js/blob/main/browser.d.ts. Or add a new definition for `HTMLMediaElement` there